### ISSUE TITLE
Accessibility on Mobile (iOS vs.12)

### ISSUE THREAD

**Issue by c-machado** [flutter/website#1574]
Hamburger menu: there isn't any option to close the menu.
https://drive.google.com/file/d/19UM_MxCeJvsA81JmO1v8PZKEopmhPda2/view?usp=sharing

Hero module: Description is not a whole paragraph, the Voiceover is announcing it by parts. 
https://drive.google.com/file/d/17zKiJgBHeJvnPgWKSa7jGlHR4r2JdTwl/view?usp=sharing

Hero module: The image in the hero module doesn't have alt text, instead of that VO is announcing some other info.
https://drive.google.com/file/d/1O-N71cE1BnBCMhgurA0zrsCKPZd6I0yQ/view?usp=sharing

Navigation: The hamburger menu is being announced even if the menu is not expanded.
https://drive.google.com/file/d/1UiMNVlPEDjWaTWJa4PdQ-WWqE8bmCJvv/view?usp=sharing

Navigation: Blocks of intro content (Fast dev, Expressive UI, Native Performance) are skip when user is navigating the content
https://drive.google.com/file/d/1zH6cV9j3amNYQoGkjxvCivQ0Nr23BT5V/view?usp=sharing

Navigation: After closing the video player (the one from the device), the user return to the page and the video popup is still visible and there is no way to close it.
![screen shot 2018-10-29 at 9 03 33 am](https://user-images.githubusercontent.com/33238142/47654788-98630a80-db59-11e8-8213-adc4d9b77f87.png)

Fast Development: icon is being announced as an image.
![screen shot 2018-10-29 at 9 09 33 am](https://user-images.githubusercontent.com/33238142/47655232-90579a80-db5a-11e8-8f97-9dab4e3c3c91.png)

Who's using Flutter: 'Google' logo doesn't have alt text.
![screen shot 2018-10-29 at 9 09 47 am](https://user-images.githubusercontent.com/33238142/47655343-d0b71880-db5a-11e8-9990-610a6082f60b.png)

Showcase Page -> Hero Carousel: the time interval for changing the content is a bit small, so the user can select the "Learn more" / "Watch the video" buttons from the next slide.
https://drive.google.com/file/d/1rT_rvN4qNiVpWyinBhNMqtImJZTgTef7/view?usp=sharing

Showcase Page -> Logos: all logos are being announced with a lot of extra information.
https://drive.google.com/file/d/1_R9uxy38Cfu9uLV7Hc9rkb5YH418Y6tM/view?usp=sharing

Reproducible on iOS 11.3.1 and 12.0.1

**Comment by sfshaza2** [comment]
I believe this is fixed.
